include/os/linux/spl/sys/debug.h

```diff
@@ -125,19 +125,27 @@ spl_assert(const char *buf, const char *file, const char *func, int line)
 		if (unlikely(!(_verify3_left OP _verify3_right)))	\
 		    spl_panic(__FILE__, __FUNCTION__, __LINE__,		\
 		    "VERIFY3(" #LEFT " "  #OP " "  #RIGHT ") "		\
-		    "failed (%px " #OP " %px)\n",			\
-		    (void *) (_verify3_left),				\
-		    (void *) (_verify3_right));				\
+		    "failed (%p " #OP " %p)\n",				\
```

Comment:
Unlike FreeBSD, where it was rightfully removed above, Linux kernel has special meaning for %px, that I suppose was used here.